Sources/Extensions/UIColor.swift

```diff
@@ -40,4 +40,35 @@ extension UIColor {
             alpha: alpha
         )
     }
+
+    func lighter(by percentage: CGFloat = 10.0) -> UIColor {
```

Comment:
Good approach but I think we do not need it for this ticket. When Willem will create new ticket then maybe we can use this approach.

Reply:
removed from this task impplementation